### ISSUE TITLE
Add rewards to GetTxBlock and GetTxBlockVerbose

### ISSUE THREAD

**Issue by maxconway** [Zilliqa/zq2#1823]
Rewards are currently hardcoded to 0, but need to be filled in with the correct value

**Comment by DrZoltanFazekas** [comment]
Nope, rewards vary per block and are only reflected in changes to the validators' reward address balances i.e. hard to reconstruct. I suggest to make a note in the docs that the rewards returned by `GetTxBlock` and `GetTxBlockVerbose` are 0.
